ocm_python_wrapper/cluster.py

```diff
@@ -249,33 +250,69 @@ def addon_info(self):
             self.addon_name
         ).to_dict()
 
-    def validate_addon_parameters(self, parameters):
+    def drop_unsupported_parameters(self, parameters=[]):
+        """ "Filters out addon parameter by conditions"""
+        filtered_parameters = []
+        for param in parameters:
+            for condition in param["conditions"]:
+                args = [x for x in condition["data"]]
+                for arg in args:
+                    value = condition["data"][arg]
+                    # print(arg+":", value)
+                    # Checking condition value with get result of cluster.instance.x.y.z... (x.y.z...=arg)
+                    result = benedict(
+                        self.instance.to_dict(), keypath_separator="."
+                    ).get(arg)
+                    if result == value or (len(value) > 1 and (result in value)):
+                        filtered_parameters.append(param)
+        return filtered_parameters
+
+    def validate_addon_parameters(self, parameters, use_api_defaults=True):
```

Comment:
`use_api_defaults` - do not set a default value here

Reply:
done